src/js/components/Button/StyledButton.js

```diff
@@ -78,8 +78,8 @@ const fillStyle = `
   flex: 1 0 auto;
 `;
 
-const plainStyle = css`
-  color: inherit;
+const plainStyle = props => css`
+  color: ${normalizeColor(props.colorValue || 'inherit', props.theme)};
```

Comment:
I'm thinking that even if the user didn't specify the prop color, we still need to have [consistent](https://github.com/grommet/grommet/blob/5c270e792a2aea3f8364f6d7bdb78b3ca121315a/src/js/components/Button/StyledButton.js#L16) behavior with how the color is handled on none plain button and apply the theme color of the button (if exists) 
```suggestion
  color: ${normalizeColor(props.colorValue || props.theme.button.plain.color || 'inherit', props.theme)};
```